buildpacks/python/python/build.go

```diff
@@ -78,16 +78,17 @@ func (b Build) Build(context libcnb.BuildContext) (libcnb.BuildResult, error) {
 		&invokerDepCacheLayer,
 		WithValidationLogger(b.Logger),
 		WithValidationFunctionClass(functionClass, isFuncDefDefault),
-		WithValidationFunctionEnvs(functionPlan.Metadata["func_yaml_envs"].(map[string]interface{})),
+		WithValidationFunctionEnvs(functionPlan.Metadata["func_yaml_envs"].(map[string]any)),
 	)
 	result.Layers = append(result.Layers, validationLayer)
 
-	for optionName, optionValue := range functionPlan.Metadata["func_yaml_options"].(map[string]interface{}) {
+	for optionName, optionValue := range functionPlan.Metadata["func_yaml_options"].(map[string]any) {
 		result.Labels = append(result.Labels, libcnb.Label{
 			Key:   optionName,
 			Value: optionValue.(string),
 		})
 	}
+	sort.Slice(result.Labels, func(i, j int) bool { return result.Labels[i].Key < result.Labels[j].Key })
```

Comment:
Introduced sorting to make the order more deterministic (also aids in testing)

Reply:
👍